src/Psalm/IssueBuffer.php

```diff
@@ -124,6 +124,41 @@ protected static function getEmacsOutput(array $issue_data)
             ($issue_data['severity'] === Config::REPORT_ERROR ? 'error' : 'warning') . ' - ' . $issue_data['message'];
     }
 
+    /**
+     * @param  array{severity: string, line_number: string, type: string, message: string, file_name: string,
+     *  file_path: string, snippet: string, from: int, to: int, snippet_from: int, snippet_to: int,
+     *  column: int} $issue_data
```

Comment:
you'll need to fix this `@param` type because I [changed](https://github.com/vimeo/psalm/commit/a99e89495b4a9fe12c1c903c326b7c85367482f1) `line_number` to `line_from` and `column` to `column_from` (and make the changes in the function too)